### PR TITLE
Add `Result` to `SLState`

### PR DESCRIPTION
**Pull request by nathanleiby**
This field is part of the states language spec, but we didn't yet support it.
I plan to use this to force a specific Result via a PASS state.

- [x] Update swagger.yml version
- [x] Run "make generate"

### CODE REVIEW

**Comment by nathanleiby** [comment]
cc @mohit

File: swagger.yml [hunk 1]
```diff
@@ -653,6 +653,8 @@ definitions:
         type: string
       ResultPath:
         type: string
+      Result:
```
Comment (nathanleiby) [inline swagger.yml]: does this need `x-omitempty: true`?

Comment (rgarcia) [reply swagger.yml]: nope, `x-omitempty` is only added to array fields where empty array `[]` is equivalent to omitting

**Comment by rgarcia** [comment]
you'll have to pull in the client change in circle-ci-integrations, e.g. https://github.com/Clever/circle-ci-integrations/pull/29

**Pull request merged**